client/homescreen/activity-panel/orders/index.js

```diff
@@ -155,20 +160,33 @@ class OrdersPanel extends Component {
 
 		const cards = [];
 		orders.forEach( ( order ) => {
```

Comment:
Not a big deal but you can destruct the order as its passed into the function:

```suggestion
		orders.forEach( ( { date_created_gmt: dateCreatedGmt } ) => {
```

Reply:
I like the idea of deconstructing `order` but I'm not sure if it will be the best solution in this case, since we need to use `order` inside the `forEach` [here](https://github.com/woocommerce/woocommerce-admin/pull/5516/files#diff-29ea5c8bb0a22385b19ffc80538ecd1a1d660f160f7a77debcb03240e62dbfe3R180) and [here](https://github.com/woocommerce/woocommerce-admin/pull/5516/files#diff-29ea5c8bb0a22385b19ffc80538ecd1a1d660f160f7a77debcb03240e62dbfe3R208). If you think it's a good idea to continue with the deconstruction please let me know and I'll change the method `orderCardTitle` and the component `OrderStatus`

Reply:
Ah sorry I didn't spot those, yep that makes sense to stay as is 😄